sdk/cosmosdb/cosmos/src/utils/globalCrypto-react-native.mts

```diff
@@ -4,5 +4,7 @@
 // isomorphic-webcrypto is not listed as a dependency in package.json because
 // doing so requires adding a bunch of react packages as peer dependencies. So,
 // it is being loaded dynamically here to not cause compiler error.
-const globalCrypto = require("isomorphic-webcrypto"); // eslint-disable-line @typescript-eslint/no-require-imports
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
I am curious why is this needed?

Reply:
Because the package isn't included as with the above text, so tsc will try to compile with it and fail.